pkg/daemon/ceph/client/crush_rule_test.go

```diff
@@ -56,7 +56,7 @@ func TestBuildCrushSteps(t *testing.T) {
 	assert.Equal(t, 4, len(steps))
 	assert.Equal(t, cephv1.DefaultCRUSHRoot, steps[0].ItemName)
 	assert.Equal(t, "datacenter", steps[1].Type)
-	assert.Equal(t, uint(2), steps[2].Number)
+	assert.Equal(t, 2, steps[2].Number)
```

Comment:
can we a unit test for negative number `-2`

Reply:
We actually don't have any code that works with the negative step numbers, we just want to allow it in case the user wants to create custom crush rules. So there is not really anything interesting for a unit test to cover.

Reply:
I thought the check is equivalent to 
`ReplicasPerFailureDomain: 2,`